backend/cmd/server/servicemanager.go

```diff
@@ -58,7 +59,9 @@ func registerServices(mux *http.ServeMux) {
 		logger.Fatal("Failed to initialize FlowMgtService", log.Error(err))
 	}
 	// TODO: this needs to be removed once the application service is refactored to use DI.
-	applicationService := appservice.GetApplicationService()
+	certservice := cert.NewCertificateService()
```

Comment:
Can we change certservice also to follow this pattern. I remember the cert service is only used within the application service.

Reply:
Cert package is refactored with https://github.com/asgardeo/thunder/pull/525
This PR depends on current #519 PR